### ISSUE TITLE
404 template does not load styles and functions

### ISSUE THREAD

**Issue by ghost** [trendschau/typemill#487]
The 404 template is actually useless at the moment.
It does not recognize twig variables from the included layout.twig file.

To reproduce:
Install Typemill in a subfolder of the domain or localhost
Try to visit a page that does not exist

![Bildschirmfoto vom 2024-06-23 15-02-37](https://github.com/typemill/typemill/assets/16516686/739a9f64-0183-4d6d-b799-2d90c5f5570c)

**Comment by trendschau** [comment]
Yes, I have to change this, I set it on the roadmap. Sorry for long waiting, but there is quite a work overload right now.

**Comment by trendschau** [comment]
This has been fixed a while ago but it is not fixed for all themes yet.
